tests/test_api_lib_utils.py

```diff
@@ -25,3 +25,11 @@ def test_01_getParam(self):
 
         v = getParam({}, "sslverify", allowed_values=["0", "1"], default="1")
         self.assertEqual("1", v)
+
+    def test_02_check_policy_name(self):
+        check_policy_name("This is a new valid Name")
+        check_policy_name("THis-is-a-valid-Name")
+        # The name "check" is reserved
+        self.assertNotEqual(ParameterError, check_policy_name, "check")
```

Comment:
Oh, this should probably be ``assertRaises``. According to codecov, we actually don't hit the error case.

Reply:
Now we shoud, right?
check this out. 
0ea6ecfd

Reply:
8c1a3ea4